modules/sr/robot3/camera.py

```diff
@@ -80,77 +55,46 @@ def parse_marker_info(model_id: str) -> MarkerInfo | None:
     object. The digits which form the 'code' are used for determining the
     properties visible in the API.
 
-    Examples: 'F00', 'F01', ..., 'B32', 'B33', ...
+    Examples: 'F00', 'F01', ...
     """
 
-    match = MARKER_MODEL_RE.match(model_id)
+    match = MARKER_MODEL_RE.match(recognition_object.getModel())
```

Comment:
Is it actually worth having this regex instead of just setting the model to the marker number?

Reply:
Maybe? It provides decoupling which allows us to convey other information here or to have things which are recognisable by some cameras but not others.